src/ui/core/Input.tsx

```diff
@@ -34,6 +34,7 @@ const StyledInput = styled(TextInput)`
   flex: 1;
   color: ${__COLORS.WHITE};
   padding: ${SPACING * 2}px 0;
+  padding-right: ${(props) => (props.status === 'valid' ? 20 : 0)}px
```

Comment:
@JonnyBurger how can i solve this linting error?

Reply:
@severinwullschleger Add a type:


```
const StyledInput = styled(TextInput)<{status: string}>``
```

Also here should be a semi colon

```suggestion
  padding-right: ${(props) => (props.status === 'valid' ? 20 : 0)}px;
```